.devcontainer/requirements.txt

```diff
@@ -18,3 +18,5 @@ ruff==0.4.6
 textalloc==1.1.3
```

Comment:
Good job @JulesL2  :fire: 

I've just introduced a profiler in this PR https://github.com/ThomasParistech/pretty-gpx/pull/20 :zap: 
Can you use it to benchmark your PR against main and provide the screenshots ?
Thanks